app/views/contact/index.html.erb

```diff
@@ -1 +1,26 @@
-<%= render 'form' %>
+<h1><%=  "SAMO"%></h1>
+<p><%= t('samo_text')%>
+<br>
+<%= t("samo_at")%>
+<%= mail_to "samo@chalmers.it"%>
+<br>
+<br>
+<h1><%= t("useful_emails")%></h1>
+<table>
+    <% ["styrIT","NollKIT","DPO","snIT","Valberedningen","digIT"].each do |c| %>
+        <% @committees.select{|e| e.name == c}.each do |com|%>
```

Comment:
This will give us a collection with only one object right?

I think it would be more idiomatic to define a `@contact_committees = Committee.where slug: ["digit", ]` in the controller, and then iterate over it in the view.